modules/installation-user-infra-machines-iso.adoc

```diff
@@ -82,6 +83,16 @@ For example, setting
 * If your system has multiple network interfaces, you can add multiple IP address configurations by specifying multiple `ip=` entries.
 * If your system has multiple network interfaces, you can use a combination of DHCP and static IP configurations. You must specify the interface to use, for example `ip=enp1s0:dhcp ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none`.
 * You can provide multiple DNS servers by adding a `nameserver=` entry for each server, such as `nameserver=1.1.1.1 nameserver=8.8.8.8`.
+<6> Optionally, you can bond multiple network interfaces to a single interface
+using the `bond=` option, as described here:
+* The syntax is: `bond=name[:slaves][:options]`.
+* In the following example, _name_ is the bonding device name (bond0), _slaves_
+represents a comma-separated list of physical (ethernet) interfaces (`em1,em2`),
+and _options_ is a comma-separated list of bonding options.
+* Note that once you
+create a bonded interface using `bond=`, you must specify how the IP address
+will be assigned (either via DHCP or via static assignment) and other
+information for the bonded interface. For example: `bond=bond0:em1,em2:mode=active-backup,tx_queues=32,downdelay=5000` then `ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none` or for DHCP: `bond=bond0:em1,em2:mode=active-backup,tx_queues=32,downdelay=5000` then `ip=bond0:dhcp`
```

Comment:
Can these examples be displayed any differently? I've read it a few times and still am not clear on how it goes together. Maybe it would help if you only used one example, if possible?

Reply:
The only other thing I can think of is breaking the ip= specific pieces into its own callout and adding that to the ip= line, how there are already 2 there.

Then you could just say that you have to make sure to update the ip= line, and only give the bond= examples? I don't know :)

Reply:
Yeah, it's messy. I'm checking into other possible ways to present this information, outside of the annotation.

Reply:
I'd use a single example for showing how to configure a bond and trim some of the options off.   

e.g  `bond=bond0:em1,em2:mode=active-backup`

Then use that bond interface in the `ip=` examples.

`ip=bond0:dhcp`

`ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none`

Reply:
Maybe make the two `ip=` examples separate bullets:

```
* To configure the bonded interface to use DHCP, use `ip=bond0:dhcp`

* To configure the bonded interface to use a static IP address, use `ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none`
````

Reply:
You're right, they need to be separated. I'm going to try moving this content into a table. It will allow me to not only keep each separate example together, but also let me mark them as code blocks.

Reply:
@miabbott @bergerhoffer I stripped down the content in the annotations and moved the content to a table. It required a bit of rewording. Let me know what you think.